arm/cookbooks/win_workstation/test/recipes/default_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'inspec'
+
+describe package 'git' do
```

Comment:
Most of these tests are described twice, here in `test/recipes` and above in `spec/unit/recipes`.
Do you see Unit vs. Spec as handling different testing scenarios?  Or will we probably just choose one?